src/util/cmdline.h

```diff
@@ -40,6 +41,46 @@ class cmdlinet
     return getoptnr(option).has_value();
   }
 
+  struct option_namest
+  {
+    option_namest(const cmdlinet &command_line);
+    struct option_names_iteratort
+      : public std::iterator<std::bidirectional_iterator_tag, std::string>
+    {
+      explicit option_names_iteratort() = default;
```

Comment:
is explicit here not just ignored?
https://stackoverflow.com/questions/6775336/c-explicit-keyword-for-function-without-arguments

Reply:
I think it is, I'll remove it.